### DISCUSSION TITLE
monitors confirguration for LO impress presentations

### DISCUSSION THREAD

**Discussion by theotrst** [hyprwm/Hyprland#3481] [Q&A]
Hi,
I am trying to set hypr.conf to correctly behave with presentations with two monitors. In LibreOffice Impress, during the slide show in the main monitor I set the presenter's view and in the external monitor the presentation. But does not work. My actual setting in hypr.conf is the simultaneous use:

```
monitor = HDMI-A-1,3440x1440,0x0,1
monitor = eDP-1,1920x1080,760x1441,1
```
(This behaves as usual: I can create new workspaces which open in the monitor I am). 
But if I use LO impress in my main monitor (eDP-1, in my case) and a launch the presentation, I see (correctly) the presenter's view in the main monitor but I **do not** see my presentation in the 2nd monitor.

 I try to set monitor mirroring in hypr.conf, but in that case LO impress "does not see" the external monitor (as it mirrors the main one).

Any idea?
thanx

t

**Comment by Elu43** [comment]
Exactly the same problem, did you find a solution ?

**Reply by theotrst** [reply]
unfortunately I do not find any solution for that.

**Comment by minsoehan** [comment]
Here, same problem. Surprisingly, no solution or workround. cyclenext or movefocus to the presentation fullscreen and move it to the workspace on external monitor manually. Inconvenient.

**Comment by Aurillium** [comment]
I don't even get presenter view while I'm presenting, but yeah, same issue here and I've seen barely anyone talking about it anywhere, and definitely no workaround

**Comment by turtureanu** [comment]
In my case, LibreOffice Impress launches in fullscreen on my main monitor, and I have to toggle full screen, in which case I get the following windows:

- Editing window of the presentation
- Window for the presenter's view (for me)
- Window for the presentation itself (for the people)

Making it behave as I would like it to requires me to manually move the presentation window to the second monitor and then fullscreen it there. The window for the presenter's view does not behave well when in fullscreen, so I have to either have it tilled or floating.
